docs/zdgbook/ObjectPublishing.rst

```diff
@@ -34,21 +34,30 @@ web publishing.
 HTTP Publishing
 ===============
 
-When you contact Zope with a web browser, your browser sends an HTTP
-request to Zope's web server. After the request is completely
-received, it is processed by 'ZPublisher', which is Zope's object
-publisher. 'ZPublisher' is a kind of light-weight ORB (Object Request
+Zope 4 no longer ships with a builtin web server, so when you want
+to interact with Zope via browser you have to setup a WSGI server as.
```

Comment:
it seems the trailing "as" should be removed.

Reply:
Thank you @ale-rt